lib/Test/Nginx/Util.pm

```diff
@@ -1168,6 +1168,11 @@ _EOC_
 $main_config
 
 http {
+    client_body_temp_path $ServRoot/client_body_temp;
+    proxy_temp_path $ServRoot/proxy_temp;
+    fastcgi_temp_path $ServRoot/fastcgi_temp;
+    scgi_temp_path $ServRoot/scgi_temp;
+    uwsgi_temp_path $ServRoot/uwsgi_temp;
```

Comment:
```suggestion
    access_log $AccLogFile; client_body_temp_path $ServRoot/client_body_temp; proxy_temp_path $ServRoot/proxy_temp; fastcgi_temp_path $ServRoot/fastcgi_temp; scgi_temp_path $ServRoot/scgi_temp; uwsgi_temp_path $ServRoot/uwsgi_temp;
```

Reply:
Some existing use cases will judge the line number of the configuration file from the Lua error messages. To maintain compatibility, it's better not to increase the number of lines in the configuration file for new configurations.

Reply:
Would you please update the PR?